blueprints/app/files/tests/test-helper.ts

```diff
@@ -5,6 +5,14 @@ import { setApplication } from '@ember/test-helpers';
 import { setup } from 'qunit-dom';
 import { start } from 'ember-qunit';
 
+window.addEventListener('beforeunload', function() {
```

Comment:
Note, and this is potentially workflow altering -- _live reload_ triggers this as well.

idk if there is a way to skip around this during live reload... but that'd be ideal.